content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -101,6 +101,7 @@ different Kubernetes components.
 | `ExperimentalHostUserNamespaceDefaulting` | `false` | Beta | 1.5 | |
 | `EvenPodsSpread` | `false` | Alpha | 1.16 | 1.17 |
 | `EvenPodsSpread` | `true` | Beta | 1.18 | |
+| `SetHostnameAsFQDN` | `false` | Alpha | 1.19 | |
```

Comment:
Hi @javidiaz .
Would it be possible to place `SetHostnameAsFQDN` feature gate after `ServiceTopology`?

Reply:
Thanks @kbhawkey. We renamed it and I forgot to move it. I changed it.